### DISCUSSION TITLE
Taxonomy for Data & Data Hardening Techniques

### DISCUSSION THREAD

**Discussion by netfl0** [d3fend/d3fend-ontology#132] [Ideas]
Potential Reference:

https://www.nist.gov/news-events/news/2019/02/methods-format-preserving-encryption-nist-requests-public-comments-draft

**Comment by netfl0** [comment]
Looking for community input on this issue, feel free to weigh in.

**Comment by kgohyeah** [comment]
Supported and suggested framework for data/data handling techniques...  Example types listed here -- this is consistent with "data element" for field level (to include unicode and form strings)

https://www.ibm.com/docs/en/iis/11.7?topic=policies-fpe-repeatable-replacement-data-masking-policy

**Comment by kgohyeah** [comment]
consider the following -- 
![data element](https://user-images.githubusercontent.com/7410184/199569793-6bbf1697-aab3-4f29-9f48-88342714769d.jpeg)

**Comment by kgohyeah** [comment]
> Looking for community input on this issue, feel free to weigh in.

\
